Missing NuGet packages for 3.4.2 release

## Describe the bug

Trying to upgrade to the 3.4.2 release, NuGet package restore fails to find the required version of two packages:

```console
  Determining projects to restore...
/home/runner/work/alexa-london-travel/alexa-london-travel/test/LondonTravel.Skill.NativeAotTests/LondonTravel.Skill.NativeAotTests.csproj : error NU1102: Unable to find package Microsoft.Testing.Extensions.CodeCoverage with version (>= 17.10.5-preview.24171.5)
/home/runner/work/alexa-london-travel/alexa-london-travel/test/LondonTravel.Skill.NativeAotTests/LondonTravel.Skill.NativeAotTests.csproj : error NU1102:   - Found 4 version(s) in NuGet [ Nearest version: 17.10.4 ]
/home/runner/work/alexa-london-travel/alexa-london-travel/test/LondonTravel.Skill.NativeAotTests/LondonTravel.Skill.NativeAotTests.csproj : error NU1102: Unable to find package Microsoft.CodeCoverage.MSBuild with version (>= 17.10.5-preview.24171.5)
/home/runner/work/alexa-london-travel/alexa-london-travel/test/LondonTravel.Skill.NativeAotTests/LondonTravel.Skill.NativeAotTests.csproj : error NU1102:   - Found 5 version(s) in NuGet [ Nearest version: 17.10.4 ]
  Restored /home/runner/work/alexa-london-travel/alexa-london-travel/src/LondonTravel.Skill/LondonTravel.Skill.csproj (in 1.71 sec).
  Failed to restore /home/runner/work/alexa-london-travel/alexa-london-travel/test/LondonTravel.Skill.NativeAotTests/LondonTravel.Skill.NativeAotTests.csproj (in 1.75 sec).
```

## Steps To Reproduce

Use version 3.4.2 of MSTest.Sdk.

## Expected behavior

Package restore succeeds.

## Actual behavior

Package restore fails.

--- a/eng/Version.Details.xml
+++ b/eng/Version.Details.xml
@@ -13,9 +13,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>2fb543a45580400a559b5ae41c96a815ea14dac5</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Testing.Extensions.CodeCoverage" Version="17.10.5-preview.24171.5">
+    <Dependency Name="Microsoft.Testing.Extensions.CodeCoverage" Version="17.10.4">
       <Uri>https://dev.azure.com/devdiv/DevDiv/_git/vs-code-coverage</Uri>
-      <Sha>577a69889b85ef831532a97e3b5e3bf8f510de4a</Sha>
+      <Sha>a12555c42042c92e5c5c12399e7ab27a3fec615c</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Testing.Platform" Version="1.2.1">
       <Uri>https://github.com/microsoft/testanywhere</Uri>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -1,12 +1,12 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <!-- This repo version -->
-    <VersionPrefix>3.4.2</VersionPrefix>
+    <VersionPrefix>3.4.3</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
   </PropertyGroup>
   <PropertyGroup Label="MSTest prod dependencies - darc updated">
     <MicrosoftDotNetBuildTasksTemplatingPackageVersion>8.0.0-beta.24225.1</MicrosoftDotNetBuildTasksTemplatingPackageVersion>
-    <MicrosoftTestingExtensionsCodeCoverageVersion>17.10.5-preview.24171.5</MicrosoftTestingExtensionsCodeCoverageVersion>
+    <MicrosoftTestingExtensionsCodeCoverageVersion>17.10.4</MicrosoftTestingExtensionsCodeCoverageVersion>
     <MicrosoftTestingPlatformVersion>1.2.1</MicrosoftTestingPlatformVersion>
     <MSTestEngineVersion>1.0.0-alpha.24271.6</MSTestEngineVersion>
   </PropertyGroup>

